features/providers/civil_application_journey.feature

```diff
@@ -294,6 +294,23 @@ Feature: Civil application journeys
     Then I click 'Save and continue'
     Then I should be on a page showing 'Check your answers'
 
+  @javascript @vcr
+  Scenario: I want to change email address from the about financial assessment page
+    Given I complete the journey as far as check your answers
+    Then I click 'Save and continue'
+    Then I am on the benefit check results page
+    Then I click 'Continue'
+    Then I should be on a page showing 'Does your client use online banking?'
+    Then I choose "Yes"
+    Then I click 'Save and continue'
+    Then I am on the About the Financial Assessment page
+    And I click Check Your Answers Change link for 'Email'
```

Comment:
You could use `Then I click link 'Change'` because there is only one `Change` link on the page.
I that case you wouldn't need the HTML to use `id="app-check-your-answers__<%= name %>">`